plugins/inputs/vsphere/README.md

```diff
@@ -17,17 +17,22 @@ vm_metric_exclude = [ "*" ]
 ```
 
 ```
-# Read metrics from one or many vCenters
 [[inputs.vsphere]]
-    ## List of vCenter URLs to be monitored. These three lines must be uncommented
-  ## and edited for the plugin to work.
-  vcenters = [ "https://vcenter.local/sdk" ]
+  ## List of vCenter URLs to be monitored.
+  vcenters = ["https://vcenter.local/sdk"]
+
+  ## vCenter Username
   username = "user@corp.local"
+
+  ## vCenter Password
   password = "secret"
 
-  ## VMs
-  ## Typical VM metrics (if omitted or empty, all metrics are collected)
-  # vm_include = [ "/*/vm/**"] # Inventory path to VMs to collect (by default all are collected)
+  ## Inventory paths for the virtual machines to gather metrics on.  If empty,
+  ## all VMs are included.
+  ##   ex: vm_include = ["/*/vm/**"]
+  # vm_include = []
```

Comment:
Make this path "/**" instead. Gets rid of that permission issue someone ran into a while ago. Requires a change to the default struct initializer in vsphere.go. See #5613